src/components/ToggleButton/ToggleButton.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import styles from './ToggleButton.module.scss';
+
+export default function ToggleButton() {
+  return (
+    <div class={styles.container}>
+      <label for="toggle-button" class={styles.text}>
+        Доход
+      </label>
+      <input type="checkbox" id="toggle-button" class={styles.ToggleButton} />
+      <label for="toggle-button" class={styles.text}>
+        Расход
+      </label>
+    </div>
+  );
+}
```

Comment:
This is only a suggestion in order to reuse this component.

```suggestion
export default function ToggleButton({ left = 'Доход', right = 'Расход' }) {
  return (
    <div class={styles.container}>
      <label for="toggle-button" class={styles.text}>
        {left}
      </label>
      <input type="checkbox" id="toggle-button" class={styles.ToggleButton} />
      <label for="toggle-button" class={styles.text}>
        {right}
      </label>
    </div>
  );
}
```